docs/upgrade.md

```diff
@@ -0,0 +1,247 @@
+# Upgrade OpenEBS
+
+This document describes the steps for upgrading the following OpenEBS reources:
+
+- [CSPC pools](#cspc-pools)
+- [cStor CSI volumes](#cstor-csi-volumes)
+
+## CSPC pools
+
+These instructions will guide you through the process of upgrading cStor CSPC pools from `1.11.0` or later to a newer release up to `1.12.0`.
```

Comment:
Q: Is there any specific reason for not supporting CSPC pool from `1.10.0` to the new Version?

Reply:
No it supports 1.10.0 as well